jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTaskListGridPresenterTest.java

```diff
@@ -69,15 +89,63 @@
     public void setupMocks() {
         //Mock that actually calls the callbacks
         callerMockTaskOperationsService = new CallerMock<TaskLifeCycleService>(taskLifeCycleServiceMock);
-        filterSettings= createTableSettingsPrototype();
+        filterSettings = createTableSettingsPrototype();
 
         when(viewMock.getListGrid()).thenReturn(extendedPagedTable);
+        when(viewMock.getVariablesTableSettings(anyString())).thenReturn(new DataSetTasksListGridViewImpl().getVariablesTableSettings("taskName"));
         when(extendedPagedTable.getPageSize()).thenReturn(10);
         when(extendedPagedTable.getColumnSortList()).thenReturn(null);
```

Comment:
null values should be the default mock value anyway, you could probably skip doing this.